patch.yaml

```diff
@@ -13,6 +13,20 @@ package:
   dependencies:
     runtime:
 
+secfixes:
+  2.7.6-r7:
```

Comment:
`epoch` needs to be bumped to make the version `2.7.6-r3`.

These patches should be acknowledged using the Wolfi version, not the Alpine version.  So, all of these should be `2.7.6-r3`.